common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -213,8 +213,10 @@ private boolean processWebsiteRequest(@NonNull final WebView view, @NonNull fina
         final String methodName = "#processWebsiteRequest";
         final PackageHelper packageHelper = new PackageHelper(getActivity().getApplicationContext());
         if (url.startsWith(AuthenticationConstants.Broker.BROWSER_DEVICE_CA_URL)
-                && packageHelper.isPackageInstalledAndEnabled(AuthenticationConstants.Broker.COMPANY_PORTAL_APP_PACKAGE_NAME)) {
-            Logger.verbose(TAG + methodName, "It is a device CA request. Company Portal is installed.");
+                && packageHelper.isPackageInstalledAndEnabled(AuthenticationConstants.Broker.IPPHONE_APP_PACKAGE_NAME)
+                && AuthenticationConstants.Broker.IPPHONE_APP_SIGNATURE.equals(
```

Comment:
Is Teams IP Phones a separate app? According to the javadoc here, it seems when the "com.microsoft.skype.teams.ipphone" installed, you should also check the company portal installation.

Reply:
Hm. Fair. The only supported scenario is their app on their phone with the CP. But I should check for both. Thank you! I'll add that back in.